go/controller/internal/controller/prautomation_controller.go

```diff
@@ -86,7 +87,7 @@ func (in *PrAutomationReconciler) Reconcile(ctx context.Context, req reconcile.R
 	if err != nil {
 		logger.Error(err, "unable to create or update prAutomation")
 		utils.MarkFalse(prAutomation.SetCondition, v1alpha1.SynchronizedConditionType, v1alpha1.SynchronizedConditionReasonError, err.Error())
-		return requeue, err
```

Comment:
Returning both non-empty `ctrl.Result` and `err` was causing a parent reconciler error. They both can't be provided at the same time.